client/my-sites/promote-post-i2/main.tsx

```diff
@@ -266,15 +258,25 @@ export default function PromotedPosts( { tab }: Props ) {
 		<Main wideLayout className="promote-post">
 			<DocumentHead title={ translate( 'Advertising - Redesign page!' ) } />
 
-			<FormattedHeader
-				brandFont
-				className="advertising__page-header"
-				headerText={ `${ translate( 'Advertising' ) } - Redesign page` }
-				subHeaderText={ campaignsData?.length ? subtitle : '' }
-				align="left"
-			/>
+			<div className="promote-post__top-bar">
```

Comment:
We need to take into account that the style of the header differs if we are showing the banner or not.
From what I can see in the designs, the font size and styling are a little different if the banner is hidden, and a description appears below saying:  `Use Blaze to grow your audience by promoting your content across Tumblr and WordPress.com.`

Reply:
Great catch, @sbarbosa! 🏆 

Fixed in https://github.com/Automattic/wp-calypso/pull/76094/commits/f72e4d87a3838bbf594d1945701a3aa00237f51a. Here's how it look like:

### With a banner
<img width="1425" alt="Page title (banner exists)" src="https://user-images.githubusercontent.com/115007291/234859826-871577a2-f77b-405c-849c-29c16cda7f64.png">

### Without a banner
<img width="1418" alt="Page title (banner is hidden)" src="https://user-images.githubusercontent.com/115007291/234859860-d8dda523-f4ab-4327-9d03-1a248e72920f.png">